frappe/core/doctype/user/test_user.py

```diff
@@ -200,16 +199,10 @@ def test_password_strength(self):
 			self.assertFalse(result.get("feedback", None))
 
 		# Test Password with Password Strenth Policy Set
-		with change_settings("System Settings", enable_password_policy=1, minimum_password_score=2):
-			# Score 1; should now fail
+		with change_settings("System Settings", enable_password_policy=1, minimum_password_score=1):
```

Comment:
Should test the failure case here still, can you try another invalid value like 5.